packages/opentelemetry-metrics/src/export/types.ts

```diff
@@ -76,6 +76,11 @@ export interface Aggregator {
   /** Updates the current with the new value. */
   update(value: number): void;
 
-  /** Returns snapshot of the current value. */
-  value(): Sum | Distribution;
+  /** Returns snapshot of the current point (value with timestamp). */
+  toPoint(): Point;
+}
+
```

Comment:
jsdoc

Reply:
I will include this in separate PR, thanks for noticing it.